coordinator/app/src/main/kotlin/net/consensys/zkevm/coordinator/app/CoordinatorApp.kt

```diff
@@ -57,9 +57,10 @@ class CoordinatorApp(private val configs: CoordinatorConfig) {
     Vertx.vertx(vertxConfig)
   }
   private val meterRegistry: MeterRegistry = BackendRegistries.getDefaultNow()
+  private val micrometerMetricsFacade = MicrometerMetricsFacade(meterRegistry, "linea")
```

Comment:
please use named arguments.

Why `linea` and not `linea.coordinator` or just empty?

Reply:
If we replace it with `linea.coordinator` or `empty` it would break the existing exported metrics, I would keep it as is, and given that we can now pass `metricsPrefix` as optional in `MicrometerMetricsFacade` constructor, we could pass a separate `metricsFacade` to class that don't need `metricsPrefix` (e.g. `VertxHttpJsonRpcClientFactory` below at line 63)